code/game/objects/items/devices/radio/radio.dm

```diff
@@ -545,23 +545,22 @@
 	set_frequency(FREQ_SYNDICATE)
 
 /obj/item/radio/borg/screwdriver_act(mob/living/user, obj/item/tool)
-	if(!keyslot)
-		to_chat(user, span_warning("This radio doesn't have any encryption keys!"))
-		return
-
-	for(var/ch_name in channels)
-		SSradio.remove_object(src, GLOB.radiochannels[ch_name])
-		secure_radio_connections[ch_name] = null
+	if(keyslot && keyslot.canRemove)
+		for(var/ch_name in channels)
+			SSradio.remove_object(src, GLOB.radiochannels[ch_name])
+			secure_radio_connections[ch_name] = null
 
-	if(keyslot)
-		var/turf/user_turf = get_turf(user)
-		if(user_turf)
-			keyslot.forceMove(user_turf)
+		if(keyslot && keyslot.canRemove)
+			user.put_in_hands(keyslot)
 			keyslot = null
 
-	recalculateChannels()
-	to_chat(user, span_notice("You pop out the encryption key in the radio."))
-	return ..()
+		recalculateChannels()
+		to_chat(user, span_notice("You pop out the encryption key in the radio."))
+		return
+	else
+		to_chat(user, span_warning("This radio doesn't have any encryption keys!"))
+		return
```

Comment:
These two returns are pointless. Also, this used to call parent, now it doesn't. Was that intentional?

Reply:
This was intentional yes, when the parent was called, you would open up the cyborgs radio as well as remove the key, but you cant access the radios wires with neither wirecutter or multitool, but if you are the cyborg, opening your radio opens up the wire window.
I dont know what the original coder of the cyborg radio had intended with it or if they missed adding something, so I decided to no longer call the parent.

the returns i will remove